backend-lib/src/main/java/cash/z/ecc/android/sdk/internal/model/IsolatedTorWalletClient.kt

```diff
@@ -0,0 +1,51 @@
+package cash.z.ecc.android.sdk.internal.model
+
+import co.electriccoin.lightwallet.client.PartialTorWalletClient
+import co.electriccoin.lightwallet.client.model.BlockHeightUnsafe
+import co.electriccoin.lightwallet.client.model.Response
+import co.electriccoin.lightwallet.client.util.use
+import kotlinx.coroutines.sync.Mutex
+import kotlinx.coroutines.sync.withLock
+
+/**
+ * A tor wallet client that uses tor connection for each individual RPC method query.
```

Comment:
My previous suggestion was altered, which changed the meaning of the sentence. Please use what I suggested previously (along with correct Tor capitalisation):
```suggestion
 * A Tor wallet client that uses an isolated Tor connection for each individual RPC method query.
```